content/python/concepts/time-module/time-module.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: 'Time Module'
+Description: 'Python has a built-in time module that has functions we can use for working with time.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Constants'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Python has a built-in `time` module that we can use for time related tasks. This module utilizes a number of different standards and objects that may be unfamiliar such as:
+
+- The epoch refers to a reference point in time from which calculations can be made. It is designated as midnight on January 1, 1970.
+- There are functions that will return values formatted using Daylight Savings Time (DST).
```

Comment:
I don't think using Daylight Savings Time would be that unfamiliar.

Reply:
It probably is relatively familiar, but it's also not a universal convention and many countries do not adjust the hour as we do. Additionally, it is a standard that is potentially in flux, there is currently a movement in the US to move permanently to DST.